charts/enterprise-logs/jsonnetfile.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I think we can ditch this and the `jsonnetfile.lock.json` for the the initial chart and revisit exports at a later point.